packages/core/project.json

```diff
@@ -31,6 +31,14 @@
         ]
       }
     },
+    "build": {
+      "executor": "@nrwl/workspace:run-commands",
+      "dependsOn": [{ "target": "prebuild", "projects": "self" }],
+      "options": {
+        "commands": [{ "command": "node tools/createLegacyTypesForCore.mjs" }],
+        "outputPath": ["dist/packages/core/types-legacy"]
+      }
+    },
```

Comment:
Runs the new script that copies all `.d.ts` files to `types-legacy` and then updates imports.